### PR TITLE
Bump and build 2.4.3

### PR DESCRIPTION
**Pull request by ianmstew**
* [x] Changelog
* [x] Bumped version numbers
* [x] Rebase after all other PRs are merged
* [x] Build

### CODE REVIEW

**Comment by ianmstew** [comment]
@ahumphreys87 @paulfalgout looks like we have a winner... agree?

File: lib/backbone.marionette.js [hunk 1]
```diff
@@ -38,7 +38,7 @@
   /* istanbul ignore next */
   // Backbone.BabySitter
   // -------------------
-  // v0.1.7
+  // v0.1.8
```
Comment (ianmstew) [inline lib/backbone.marionette.js]: Hmm, why is babysitter not 0.1.10?  Ahhh.  I think I know.

**Comment by paulfalgout** [comment]
:+1:  finally :-)

**Comment by jasonLaster** [comment]
@ianmstew - mind changing this PR to be against MASTER. That way the branch is truly representative of what will land. We should see all of the commits for example, including the merge commits.

**Comment by ianmstew** [comment]
Moved to #2740

**Pull request closed without merging**